Uplift/Controllers/FilterViewController.swift

```diff
@@ -459,7 +459,7 @@ class FilterViewController: UIViewController, RangeSeekSliderDelegate {
 
         if classTypeDropdownData.dropStatus == .half || classTypeDropdownData.dropStatus == .down {
             (classTypeDropdown.headerView(forSection: 0) as! DropdownHeaderView).downArrow.image = .none
-            (classTypeDropdown.headerView(forSection: 0) as! DropdownHeaderView).rightArrow.image = #imageLiteral(resourceName: "right_arrow")
+            (classTypeDropdown.headerView(forSection: 0) as! DropdownHeaderView).rightArrow.image = UIImage(named: "right_arrow")
```

Comment:
Rather than doing `UIImage(named:` and then the image's file name, could you put the file names in `Constants.swift -> ImageNames`?  Same goes for the other changes.

Reply:
I think its only necessary to put it into a `Constants` file if the imageName is used in multiple places